### PR TITLE
configure notification lifetime

### PR DESCRIPTION
**Pull request by keichwa**
https://github.com/SUSE/doc-susemanager/issues/256
https://github.com/SUSE/spacewalk/issues/6439

back-port to 3.2 when approved.

### CODE REVIEW

File: adoc/reference-webui-overview.adoc [hunk 1]
```diff
@@ -99,6 +99,13 @@ The menu:Main Menu[Home > Notification Messages] page allows you to manage your
 
 image::home_notification_messages.png[scaledwidth=80%]
 
+You can configure the notifications lifetime in [path]``/etc/rhn/rhn.conf`` (the default lifetime is 30 days):
```
Comment (Loquacity) [inline adoc/reference-webui-overview.adoc]: Hrm. What does "lifetime" mean in this situation? Is it how long the notification remains visible for until it is manually cleared? Or something else? I'd like this to be more specific.

Comment (Loquacity) [reply adoc/reference-webui-overview.adoc]: So the wording should probably be something like this:

`You can configure the amount of time before notifications are deleted in [path]``/etc/rhn/rhn.conf`` (the default lifetime is 30 days, after which notifications are deleted from the database, regardless of read status):`

Comment (Loquacity) [reply adoc/reference-webui-overview.adoc]: I don't really like the parenthesis, but I'll leave that up to @keichwa to determine, based on context.

**Comment by keichwa** [comment]
thanks for all your input.  please check again.

**Pull request merged**